temporalcli/commandsmd/commands.md

```diff
@@ -1,487 +1,1185 @@
+* `--retention` (duration) -
+  Length of time a closed Workflow is preserved before deletion
+* `--visibility-archival-state` (string-enum) -
+  Visibility archival state. Options: disabled, enabled
+* `--visibility-uri` (string) -
+  Archive visibility data to this `URI`.
+  Once enabled, cannot be changed.
+
+### temporal operator search-attribute: Search Attribute operations
+
+Create, list, or remove Search Attributes, fields stored in a Workflow
+Execution's metadata:
 
-### temporal operator search-attribute: Operations applying to Search Attributes
+```
+temporal operator search-attribute create \
+    --name YourAttributeName \
+    --type Keyword
+```
 
-Search Attribute commands enable operations for the creation, listing, and removal of Search Attributes.
+Supported types include: Text, Keyword, Int, Double, Bool, Datetime, and
+KeywordList.
 
-### temporal operator search-attribute create: Adds one or more custom Search Attributes
+### temporal operator search-attribute create: Add custom Search Attributes
 
-`temporal operator search-attribute create` command adds one or more custom Search Attributes.
+Add one or more custom Search Attributes:
+
+```
+temporal operator search-attribute create \
+    --name YourAttributeName \
+    --type Keyword
+```
 
 #### Options
 
-* `--name` (string[]) - Search Attribute name. Required.
-* `--type` (string[]) - Search Attribute type. Options: Text, Keyword, Int, Double, Bool, Datetime, KeywordList. Required.
+* `--name` (string[]) -
+  Search Attribute name. Required
+* `--type` (string[]) -
+  Search Attribute type.
+  Options: Text, Keyword, Int, Double, Bool, Datetime, KeywordList.
+  Required.
 
-### temporal operator search-attribute list: Lists all Search Attributes that can be used in list Workflow Queries
+### temporal operator search-attribute list: List Search Attributes
 
-`temporal operator search-attribute list` displays a list of all Search Attributes.
+Display a list of active Search Attributes that can be assigned or used with 
+Workflow Queries.
 
-### temporal operator search-attribute remove: Removes custom search attribute metadata only
+```
+temporal operator search-attribute list
+```
+
+### temporal operator search-attribute remove: Remove custom Search Attributes
+
+Remove custom Search Attributes from the options that can be assigned or
+used with Workflow Queries.
+
+```
+temporal operator search-attribute remove \
+    --name YourAttributeName
+```
 
-`temporal operator search-attribute remove` command removes custom Search Attribute metadata.
+Remove without confirmation:
+
+```
+temporal operator search-attribute remove \
+    --name YourAttributeName \
+    --yes false
+```
 
 #### Options
 
-* `--name` (string[]) - Search Attribute name. Required.
-* `--yes`, `-y` (bool) - Confirm prompt to perform deletion.
+* `--name` (string[]) -
+  Search Attribute name.
+  Required.
+* `--yes`, `-y` (bool) -
+  Override confirmation request before removal.
```

Comment:
```suggestion
  Don't prompt to confirm removal.
```

Reply:
Adopted

Reply:
And did a global audit to bring everything into line.